macros/src/main/scala/me/rexim/morganey/meta/QuotationMacro.scala

```diff
@@ -0,0 +1,161 @@
+package me.rexim.morganey.meta
+
+import me.rexim.morganey.ast._
+
+import scala.collection.mutable.ArrayBuffer
+import scala.reflect.macros.whitebox.Context
+
+private[meta] class QuotationMacro(val c: Context) {
```

Comment:
Yeah, I'm gonna create a separate issue and we will discuss this later when the time is right.